### PR TITLE
MC-10287: Added documentation of the product catalog and pricing

### PR DESCRIPTION
**Pull request by fyrnaga**
#### Changes made

- Added the product catalog api documentation for list and retrieve
- Added the pricing api documentation for list and retrieve

#### Related PRs
- []()

### CODE REVIEW

File: source/includes/administration/_pricings.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
+### Pricings
+
+The pricings determines the price for each product defines for a selected catalogs.
```
Comment (lamminade) [inline source/includes/administration/_pricings.md]:
```suggestion
The pricing determines the price for each product defined for a selected catalog.
```

File: source/includes/administration/_pricings.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      "effectiveDate": "2020-05-08T16:53:45Z"
+    }
+  ]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the pricing.
+`name`<br/>*Object* | The name object in each language for the pricing (en, fr or es).
```
Comment (lamminade) [inline source/includes/administration/_pricings.md]:
```suggestion
`name`<br/>*Object* | The name object in each language for the pricing (en, fr, or es).
```

File: source/includes/administration/_pricings.md [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+`description`<br/>*string* | The description object in each language for the pricing (en, fr or es).
```
Comment (lamminade) [inline source/includes/administration/_pricings.md]:
```suggestion
`description`<br/>*string* | The description object in each language for the pricing (en, fr, or es).
```

File: source/includes/administration/_pricings.md [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+`supportedCurrencies`<br/>*Array[string]* | List of the currency which are link to this pricing.
```
Comment (lamminade) [inline source/includes/administration/_pricings.md]:
```suggestion
`supportedCurrencies`<br/>*Array[string]* | List of the currencies which are supported by this pricing.
```

File: source/includes/administration/_pricings.md [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+`productCatalogs`<br/>*Object* | List of product catalog object which are attached to this pricing.
```
Comment (lamminade) [inline source/includes/administration/_pricings.md]:
```suggestion
`productCatalogs`<br/>*Object* | List of product catalog objects which are attached to this pricing.
```

File: source/includes/administration/_products.md [hunk 6]
```diff
@@ -0,0 +1,207 @@
+### Product Catalogs
+
+The product catalogs determines the product configured for a service type and maybe specific connections.
```
Comment (lamminade) [inline source/includes/administration/_products.md]:
```suggestion
The product catalogs determine the product configured for a service type and optionally for specific connections.
```

File: source/includes/administration/_pricings.md [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+`organization.id`<br/>*UUID* | UUID of the organisation to which belongs the pricing.
+`pricingProducts`<br/>*Array[Object]* | The list of products assigned to the pricing.
```
Comment (lamminade) [inline source/includes/administration/_pricings.md]:
```suggestion
`pricingProducts`<br/>*Array[Object]* | The list of product pricings assigned to the pricing.
```
? would this maybe be more clear

File: source/includes/administration/_pricings.md [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+      "en": "descen",
+      "fr": "testfr"
+    },
+    "id": "096bbfee-a6e6-4373-8a4e-1a8b0f2bf484",
+    "effectiveDate": "2020-05-08T16:53:45Z"
+  }
+}
+```
+
+Attributes | &nbsp;
```
Comment (lamminade) [inline source/includes/administration/_pricings.md:220]: the same comments as above

File: source/includes/administration/_products.md [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the product catalog.
+`name`<br/>*Object* | The name object in each language for the product catalog (en, fr or es).
+`description`<br/>*string* | The description object in each language for the product catalog (en, fr or es).
+`mode`<br/>*string* | Identify the mode if it is for all service type or specific connections. Possible values: ALL_CONNECTIONS_OF_TYPE, SPECIFIC_CONNECTIONS.
+`serviceType`<br/>*Object* | The service connection type to which is bound the product catalog.
```
Comment (lamminade) [inline source/includes/administration/_products.md]:
```suggestion
`serviceType`<br/>*Object* | The service connection type the product catalog is bound to.
```

File: source/includes/administration/_products.md [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+`connectionIds`<br/>*Array[UUID]* | Array of the UUID service connection to which the catalog is bound to.
```
Comment (lamminade) [inline source/includes/administration/_products.md]:
```suggestion
`connectionIds`<br/>*Array[UUID]* | Array of the UUID service connection the catalog is bound to.
```

File: source/includes/administration/_products.md [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+`changes`<br/>*Array[Object]* | Array of all changes done on the product catalog.
+`categories`<br/>*Array[Object]* | The list of product categories object.
+`categories.id`<br/>*Array[Object]* | The list of product categories object.
```
Comment (lamminade) [inline source/includes/administration/_products.md]:
```suggestion
`categories.id`<br/>*Array[Object]* | The id of product category object.
```

File: source/includes/administration/_products.md [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+      ]
+    }
+  ]
+}
+```
+
+Attributes | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The UUID of the product catalog.
+`name`<br/>*Object* | The name object in each language for the product catalog (en, fr or es).
```
Comment (lamminade) [inline source/includes/administration/_products.md]:
```suggestion
`name`<br/>*Object* | The name object in each language for the product catalog (en, fr, or es).
```
technically correct without the comma, i just think its more explicitly clear

**Review by lamminade — APPROVED** [review]
Looks good, some small comments

File: source/includes/administration/_pricings.md (hunk 2 above)
Comment (evanluc) [reply source/includes/administration/_pricings.md]: We probably shouldn't hardcode the languages because they vary per branding.

Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: How should i describe it then? Just removing the parenthesis seems not complete

File: source/includes/administration/_pricings.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_pricings.md (hunk 2 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_pricings.md (hunk 4 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_pricings.md (hunk 5 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_pricings.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_pricings.md (hunk 8 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md:220]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_products.md (hunk 6 above)
Comment (fyrnaga) [reply source/includes/administration/_products.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_pricings.md (hunk 7 above)
Comment (fyrnaga) [reply source/includes/administration/_pricings.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_products.md (hunk 9 above)
Comment (fyrnaga) [reply source/includes/administration/_products.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_products.md (hunk 10 above)
Comment (fyrnaga) [reply source/includes/administration/_products.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_products.md (hunk 12 above)
Comment (fyrnaga) [reply source/includes/administration/_products.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

File: source/includes/administration/_products.md (hunk 11 above)
Comment (fyrnaga) [reply source/includes/administration/_products.md]: f428ed5b76ae49b8bf4e644a9c5cd49c51399110

**Review by evanluc — APPROVED** [review]

**Pull request merged**